Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs

```diff
@@ -288,6 +292,8 @@ public Vector3 GrabPoint
         private Quaternion startObjectRotationFlatCameraSpace;
         private Quaternion hostWorldRotationOnManipulationStart;
 
+        private FixedDistanceConstraint moveConstraint;
+        private RotationAxisConstraint rotateConstraint;
         private TransformScaleHandler scaleHandler;
```

Comment:
should we change this to be ScaleConstraint? would make more sense I suppose / less confusing for users - (for later: we can create a doc chapter about all our constraints and how they're working together with different ux elements)

Reply:
I changed it to ScaleConstraint initially, but it broke references to it in the example scenes. Is there a way to rename a component without breaking references?

Reply:
It looks like it's worked now. I may have tried to move it as well last time and that's why it didn't work. I've gone with `MinMaxScaleConstraint`.

Reply:
I would've preferred ScaleConstraint - i don't think the minmax is necessary?